fix ppa repositories in ci_build

jendap opened the pull request:
Those repositories are just workaround for old versions of libraries on old ubuntu (trusty). Those ppas do not make sense on newer ubuntu or debian.

jendap committed:
fix ppa repositories in ci_build

martinwicke commented on tensorflow/tools/ci_build/Dockerfile.android:
```diff
@@ -5,7 +5,9 @@ MAINTAINER Jan Prach <jendap@google.com>
 COPY install/*.sh /install/
 RUN /install/install_bootstrap_deb_packages.sh
-RUN add-apt-repository -y ppa:openjdk-r/ppa
+RUN add-apt-repository -y ppa:openjdk-r/ppa \
```
This should be three different run lines, or don't repeat the add-apt-repository -y. 

See tests.

martinwicke committed:
Make diag_op_test faster

Removes high rank and large tensors from the test.

caisq committed:
Bumping version: 0.7.1 --> 0.8.0rc0 (#1884)

version_test.py: patch can be alphanumeric (e.g., 0rc0)

shancarter committed:
Adding link to playground (#1887)

martinwicke committed:
Make diag_op_test faster (#1889)

Removes high rank and large tensors from the test.

mrry committed:
Fix bugs in the indexing operator with latest NumPy (#1895)

* Made the indexing operator accept objects that can be converted with `int()`.

Previously, the `_SliceHelper()` would test for `isinstance(..., int)`, which
is too restrictive.

* Replace `np.random.random_integers()` with `np.random.randint()`

martinwicke committed:
Make learn:test_base medium size

martinwicke committed:
Merge pull request #1896 from martinwicke/master

Fixes for release

martinwicke committed:
Make sparse_tensor_dense_matmul_op_test medium

martinwicke committed:
Merge pull request #1897 from martinwicke/master

Make sparse_tensor_dense_matmul_op_test medium

martinwicke committed:
R0.8 Release notes (#1898)

Add 0.8 release notes

martinwicke committed:
Update roadmap.md (#1900)

jendap commented:
I'm closing this one. I will reopen it against r0.8 branch.
